Adding link to permissions (#7092)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/billing.md

### Usage notifications 

> Every plan automatically sends email alerts when 75%, 90%, and 100% of usage estimates have been reached. In the Team plan, all users within the Owner group will receive alerts. In Enterprise plans, all users with the Account Admin and Billing Admin [permission sets](/docs/cloud/manage-access/enterprise-permissions) will receive alerts. Users cannot opt out of these emails. If you would like additional users to receive these alert emails, please provide them with the applicable permissions mentioned above. Note that your usage may already be higher than the percentage indicated in the alert due to your usage pattern and minor latency times. 

Reviewer: i know its not within the scope but sugg breaking down the chunky paragraph into bullets for easier reading
```suggestion
Every plan automatically sends email alerts when 75%, 90%, and 100% of usage estimates have been reached.
- Team plan &mdash; All users within the Owner group receive alerts. 
- Enterprise plans &mdash; All users with the Account Admin and Billing Admin [permission sets](/docs/cloud/manage-access/enterprise-permissions#permission-sets) receive alerts. 

Users cannot opt out of these emails. To have additional users to receive these alert emails, assign them the applicable permissions mentioned earlier. Note that your usage may already be higher than the percentage indicated in the alert due to your usage pattern and minor latency times.
```
